tgui/packages/tgui/interfaces/Keybinds/formatKeyboardEvent.ts

```diff
@@ -44,8 +43,8 @@ export const formatKeyboardEvent = (event: React.KeyboardEvent): string => {
 
   if (isStandardKey(event)) {
     const key = event.key.toUpperCase();
-    text += KEY_CODE_TO_BYOND[key] || key;
+    text += KEY_CODE_TO_BYOND[key] || key === ' ' ? 'Space' : key;
```

Comment:
Is it not possible to include a `' ': 'Space'` in the `KEY_CODE_TO_BYOND` map? Notably, there's this as the reverse: https://github.com/tgstation/tgui-core/commit/95608933d9248e7b91506ed365b5049ff0164bd6#L45

Reply:
I thought it wasn't, but turns out it is possible!